packages/docs/src/page-configs/ui-elements/data-table/examples/CustomSlots.vue

```diff
@@ -19,8 +20,8 @@ export default defineComponent({
         id: 1,
         name: 'Leanne Graham',
         username: 'Bret',
-        email: 'Sincere@april.biz',
         address: {
+          email: 'Sincere@april.biz',
```

Comment:
Better not to touch the original example data.

Reply:
Restored.